src/main/java/harvard/TaskList.java

```diff
@@ -0,0 +1,148 @@
+package harvard;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import harvard.tasks.Task;
+import harvard.tasks.Event;
+import harvard.tasks.Deadline;
+import harvard.tasks.Todo;
+import harvard.exceptions.HarvardException;
+
+/**
+ * The TaskList class represents a list of tasks in the Harvard application.
+ * It provides methods to manipulate the task list, such as adding, deleting, and marking tasks.
+ */
+public class TaskList {
+
+    /**
+     * The list that stores tasks.
+     */
+    private List<Task> taskList = new ArrayList<>();
```

Comment:
Would it be more readable to use a plural term like `tasks` to represent the list of tasks?